Common/tests/Pollster/DataPrefetcherTest.cs

```diff
@@ -393,6 +396,7 @@ public async Task EmptyPayloadAndOneDependencyWithDataStateMachine()
                                                                   DateTime.Now,
                                                                   new Output(true,
                                                                              "")),
+                                                     "/socket",
```

Comment:
inconsistent formating

Reply:
Actually, it is the right one !

Reply:
Ok; the ones above are not formated like this

Reply:
My bad, there was a mistake there. It is fixed now